pyinfra/modules/iptables.py

```diff
@@ -0,0 +1,65 @@
+# pyinfra
+# File: pyinfra/modules/iptables.py
+# Desc: manage iptables rules
+
+'''
+The iptables modules handles iptables rules
+'''
+
+from __future__ import unicode_literals
+
+from pyinfra.api import operation
+
+
+@operation
+def forward(state, host, dport, to_ip, to_port=None,
+            interface='eth0', protocol='tcp', chain='PREROUTING',
+            present=True):
+    '''
+    Manage an iptables rule to forward packets arriving on a given port and
+    interface to another host.
+
+    + dport: port on which the packets are arriving
+    + to_ip: ip of the destination
+    + to_port: port of the destination (default to dport)
+    + interface: interface on which the packets are arriving
+    + protocol: protocol (tcp or udp)
+    + chain: iptables chain (PREROUTING, POSTROUTING, OUTPUT)
+    + present: whether the rule should be present or removed
+    '''
+    to_port = to_port or dport
+    definition = {
+        'dport': str(dport),
+        'destination': '{}:{}'.format(to_ip, to_port),
+        'interface': interface,
+        'protocol': protocol,
+        'match': protocol,
+        'chain': chain,
+        'jump': 'DNAT',
```

Comment:
Can we also make this a kwarg? That should mean handling most NAT rules is possible with this operation :)